pkg/kritis/cryptolib/verifier_test.go

```diff
@@ -105,6 +108,12 @@ func (v testVerifierFail) verifyPkix([]byte, []byte, []byte) error {
 	return errors.New("error verifying PKIX")
 }
 
+type testAuthAttFormer struct{}
+
+func (f testAuthAttFormer) formAuthenticatedAttestation(payload []byte) (*authenticatedAttestation, error) {
+	return nil, nil
```

Comment:
Even in test code, returning nil value with non-nil error sets off red flags. hard-code a minimal valid authenticatedAttestation instead

Reply:
Done.